site/nav.yml

```diff
@@ -43,21 +43,20 @@ nav:
   - Project: 
     - Community: community.md
     - Contributing: contribute.md
-    - REST Catalog Spec: https://editor-next.swagger.io/?url=https://raw.githubusercontent.com/apache/iceberg/main/open-api/rest-catalog-open-api.yaml
-    - Table Spec: spec.md
-    - View spec: view-spec.md
-    - Puffin spec: puffin-spec.md
-    - AES GCM Stream spec: gcm-stream-spec.md
-    - Implementation status: status.md
     - Multi-engine support: multi-engine-support.md
     - How to release: how-to-release.md
-    - Terms: terms.md
     - ASF:
       - Sponsorship: https://www.apache.org/foundation/thanks.html
       - Events: https://www.apache.org/events/current-event.html
       - Privacy: https://privacy.apache.org/policies/privacy-policy-public.html
       - License: https://www.apache.org/licenses/
       - Security: https://www.apache.org/security/
       - Sponsors: https://www.apache.org/foundation/thanks.html
-  - Concepts: 
-    - Catalogs: concepts/catalog.md
+  - Specification:
+    - Table Spec: spec.md
+    - View spec: view-spec.md
+    - Puffin spec: puffin-spec.md
+    - AES GCM Stream spec: gcm-stream-spec.md
+    - REST Catalog Spec: https://editor-next.swagger.io/?url=https://raw.githubusercontent.com/apache/iceberg/main/open-api/rest-catalog-open-api.yaml
+    - Implementation status: status.md
+    - Terms: terms.md
```

Comment:
I would also be open to bringing this back to its original place. Now it is at the very bottom of the specs, but I think it is nice for folks to understand the concepts without plowing through the specs themselves. Curious to learn what others think.

Reply:
Do you mean keeping it at `Project` tab while merging `Catalog` into it?
Actually, these terms are more or less explained in the table spec. Maybe it's valuable as additional references as part of `Specification`?

Reply:
maybe Fokko meant moving the `Terms` page at the top of the `Specification` section. that would make sense to me.

Reply:
Top sounds good to me